cross-wiki-weekend-2021.php

```diff
@@ -0,0 +1,296 @@
+            </ul>
+            <li>As long as you make at least one non-minor edit, you're eligible to win the prize.</li>
+            <li>Each wiki is a bit different! Become familiar with the differing guidelines, styling, and policies of each wiki. Reach out to each wiki's help resources if you have any questions.</li>
+            <li>At the end of the event, fill out and submit the participation form below by <b>September 30<sup>th</sup></b> for a chance to win the prize. Even if you don't want the prize, we appreciate you filling the form out for statistical purposes.</li>
+        </ul>
+
+        <br>
+
+        <a name="guide"></a>
+        <h1 id="Editing Guide">Editing Guides</h1>
+
+        Below is a list of participating wikis and links to to-do lists and guides.
+        Some wikis have created guides just for this event, which have been linked in bold.
+        If you're unsure of where to begin, you'll hopefully find these links useful!
+        Take a moment to familiarize yourself with each wiki's rules and customs before diving in.
+
+        <br>
+
+        <h3 class="text-center">English Wikis</h3>
+        <div class="text-center cww-grid">
+            <?php
+            $wikis = $dataHelper->getMemberWikis('en');
+            foreach ($dataHelper->getCWW('2021', 'en') as $cww) {
+                $id = $cww->id;
+                $wiki = $wikis[array_search($id, array_column($wikis, 'id'))];
+                echo "
+                    <div>
+                        <a href='{$dataHelper->getWikiLink($wiki->url, $cww->page)}'>
+                            <img class='margin-auto' src='{$wiki->logo}' alt='{$wiki->title}' width='100' />
+                            <br />
+                            {$cww->title}
+                        </a>
+                    </div>
+                ";
+            }
+            ?>
+        </div>
+        
+        <h3 class="text-center">Italian Wikis</h3>
+        <div class="text-center cww-grid">
+            <?php
+            $wikis = $dataHelper->getMemberWikis('it');
+            foreach ($dataHelper->getCWW('2021', 'it') as $cww) {
+                $id = $cww->id;
+                $wiki = $wikis[array_search($id, array_column($wikis, 'id'))];
+                echo "
+                    <div>
+                        <a href='{$dataHelper->getWikiLink($wiki->url, $cww->page)}'>
+                            <img class='margin-auto' src='{$wiki->logo}' alt='{$wiki->title}' width='100' />
+                            <br />
+                            {$cww->title}
+                        </a>
+                    </div>
+                ";
+            }
+            ?>
+        </div>
+
+        <br><br>
+
+        <h1 id="Drawing">Drawing</h1>
+        The winner of the contest is determined by random draw.
+        Each person's contributions are rated on a scale of 1 to 5; the more substantial one's contributions, the greater the chance of winning.
```

Comment:
Above mentions 1 to 10, here mentions 1 to 5.

Reply:
Thanks!